pkg/policyengine/PolicyDispatcher.go

```diff
@@ -248,19 +236,10 @@ func (pH *PolicyHandler) DeleteLBPolicy(policy *api.Policy) error {
 	return pH.loadBalancer.DeletePolicy(lbPolicy)
 }
 
-func (pH *PolicyHandler) AddAccessPolicy(policy *api.Policy) error {
-	connPolicy, err := connPolicyFromBlob(policy.Spec.Blob)
-	if err != nil {
-		return err
-	}
-	return pH.connectivityPDP.AddOrUpdatePolicy(connPolicy)
+func (pH *PolicyHandler) AddAccessPolicy(policy *crds.AccessPolicy, privileged bool) error {
```

Comment:
nit: keep variable lower case as you're not using full words anyway? (`ph *PolicyHandler` works for me the same as `pH *PolicyHandler`)